app/models/article.rb

```diff
@@ -0,0 +1,10 @@
+class Article < ApplicationRecord
+    validates :title, presence: true,
+                    length: {minimum: 5}
```

Comment:
大抵`{}`の内側にスペースを1つ入れますね。あと、この行はあまり長くないので1行にしてしまっていいと思います。
```suggestion
validates :title, presence: true, length: { minimum: 5 }
```

Reply:
美しいです・・・。修正します。